suites/reef/nvmeof/tier-2_nvmeof_gateway_operations.yaml

```diff
@@ -300,6 +300,18 @@ tests:
               args:
                 subsystem: nqn.2016-06.io.spdk:cnode1
                 nsid: 2
+          - config:
+              service: namespace
+              command: delete           # namespace delete
+              args:
+                subsystem: nqn.2016-06.io.spdk:cnode1
+                nsid: 3
+          - config:
+              service: namespace
+              command: delete           # namespace delete
+              args:
+                subsystem: nqn.2016-06.io.spdk:cnode1
+                nsid: 4
           - config:
               service: subsystem
               command: delete           # subsystem delete
```

Comment:
IMO, its better to have a check on gw info after subsystem deletion based on https://bugzilla.redhat.com/show_bug.cgi?id=2308160

Reply:
Sure

Reply:
Done.